plugin/tokudb-backup-plugin/tokudb_backup.cc

```diff
@@ -3,6 +3,7 @@
 #ident "Copyright (c) 2014 Tokutek Inc.  All rights reserved."
 
 #define MYSQL_SERVER
+#define HAVE_REPLICATION
```

Comment:
Well, actually, it's not required for 5.6, but it's required for 5.7. But would really prefer to have as more similar code as possible for 5.6 and 5.7 to avoid merge conflicts. Otherwise I would not use vector to gather slave information in 5.6 because there multisource replication is not implemented for 5.6.